poc/vidpf.py

```diff
@@ -361,7 +371,7 @@ def convert(self,
         xof = XofFixedKeyAes128(seed, dst(ctx, USAGE_CONVERT), nonce)
         next_seed = xof.next(XofFixedKeyAes128.SEED_SIZE)
         payload = xof.next_vec(self.field, self.VALUE_LEN)
-        return (next_seed, payload)
+        return (next_seed, self.transform(payload, level))
```

Comment:
I'm wondering if it's a good idea to transform the derived payload. An alternative would be to compute the length of the transformed payload and simply generate that many field elements here.

Reply:
I prefer it as it is now that we're calling transform.

Reply:
That was my concern as well, though I think we're OK for the current set of Mastic variants, and probably any "natural" variant for which the output share is meaningfully secret, i.e, not just a constant.

The reason I didn't initially go with the length is that it adds a lot of code complexity to libprio.

Reply:
(This change is much more invasive if have to deal with correction word decoding, which we don't here)